AI/Nullkiller/AIGateway.cpp

```diff
@@ -23,8 +23,15 @@
 #include "AIGateway.h"
 #include "Goals/Goals.h"
 
+VCMI_LIB_NAMESPACE_BEGIN
+
 class CGVisitableOPW;
 
+VCMI_LIB_NAMESPACE_END
```

Comment:
not sure why this is needed - `using namespace` is already in StdInc.h

Reply:
Cleared some unused forward declarations